### ISSUE TITLE
Remove Code of Conduct files and references

### ISSUE THREAD

**Issue by tilleryd** [DecipherNow/helm-charts#288]
1. Ensure that Code of Conduct files are deleted in the root or `/docs` folders. 
2. Delete any reference to Code of Conduct in each project's README (typically at the bottom).

Repos:

- gm-control-docs,
- gm-control-cli,
- gm-data
- gm-data-ui
- gm-control-nonstdlib
- gm-control-test
- gm-control-stats
- gm-control-cache
- gm-control-codec
- gm-control-idgen
- gm-control

**Comment by alecholmez** [comment]
gm-data and gm-data-ui do not have any references to Code Of Conduct anywhere

**Comment by chrisbsmith** [comment]
@tilleryd I don't think this is a helm-charts issue.  Maybe it was created from Zenhub and the repo wasn't changed?
